index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Scheherazade+New&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+    <title>African Devs</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="nav-buttons">
+
+                <ul>
+                    
+                    <li>
+                        <a href="#projects">Projects</a>
+                    </li>
+                    <li>
+                        <a href="#skills">Skills</a>
+                    </li>
+                    <li>
+                        <a href="#contacts">Contacts</a>
+                    </li>
+    
+                </ul>
+            </div>
+
+            <div id="github_image">
+                
+                <a target="_blank" id="profile-link" href="https://github.com/matovu-farid/microverse_maxwell_farid/blob/85fcd846190f554111ebfb836d051ce16a43092b/index.html">
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Octicons-mark-github.svg/1200px-Octicons-mark-github.svg.png" alt="Github">
+                    Github
+                </a>
+            </div>
+        </nav>
+    </header>
+    <section id="welcome-section">
+        <h1>
+            Welcome to my Portfolio
+        </h1>
+
+    </section>
+    <section id="projects">
+        <div>
+            <h1>Projects</h1>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut possimus saepe ullam ipsa. Repellendus officiis exercitationem dolor nobis ea voluptatibus obcaecati, sit minus. Placeat, ut perspiciatis maxime aliquam iste rem.
+        </div>
+
+      <div class="project-tile">
+        <h2 class="project-title">
+            Restaurant Website
+        </h2>
+        <img src="https://imaxxiportfolio.netlify.app/assets/restau.JPG" class="restau-imgs" alt="project">
+        <img src="https://imaxxiportfolio.netlify.app/assets/restau_2.JPG" class="restau-imgs" alt="project">
+        
+        <p class="description">
+            A restaurant advertisement and order placement website. 
+            Come one, come all ! Get access to special delicasies with ease.
+            <br>
+            <a href="https://restaurantconfusion.netlify.app">Check it out</a>
+            
+            
+        </p>
+      </div>
+
+       
+      <div class="project-tile">
```

Comment:
Please use a section tag "

" instead of a "
". This will give semantic meaning to your work and will be used to convey the structure of the document in a clear manner.
```suggestion
      <section class="project-tile">
```